os-ui/src/component/Navbar-Component/NavbarOnSignIn/NavbarOnSignIn.tsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+import styles from './NavbarOnSignIn.module.scss';
+import ArrowDownIcon from '../../../assets/svg/Arrow-down.svg';
+import NotificationIcon from '../../../assets/svg/Notification.svg';
+import BookmarkIcon from '../../../icons/Bookmark';
+import { SignoutIcon } from '../../../icons/Signout/Signout';
+import ProfilePortalContent from '../../ProfilePortalContent/ProfilePortalContent';
+import { Portal } from '../../Portal/Portal';
+
+/* eslint-disable max-len */
+
+const NavbarOnSignIn = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const navigate = useNavigate();
+  const { t } = useTranslation();
+  const {
+    mainContent, navMainContent, userInfoContent, bookmarkIconStyle, notificationIcon, avatarLogo, arrowDownIcon,
+  } = styles;
+
+  const handleProfilePortal = () => {
+    setIsOpen((prevState) => !prevState);
+  };
+
+  return (
+    <nav className={mainContent}>
+      <h2 onClick={() => navigate('/')}>{t('string.homePage.navBar.logo')}</h2>
+      <div className={navMainContent}>
+        <p onClick={() => navigate('/exploreLearningPaths')}>{t('string.homePage.navBar.allLearningPaths')}</p>
+        <p onClick={() => navigate('/myLearningPath')}>{t('string.homePage.navBar.myLearningPaths')}</p>
+        <p onClick={() => navigate('/mentors')}>{t('string.homePage.navBar.mentors')}</p>
+        <div className={bookmarkIconStyle}>
+          <BookmarkIcon iconSize="20px" />
```

Comment:
is it ok to hardcode the size in pixels ?

Reply:
iconSize is a prop which is the value of the font size of style attribute inside the BookmarkIcon component